website/pages/api-docs/connect/intentions.mdx

```diff
@@ -427,6 +620,19 @@ $ curl \
 
 ## Check Intention Result
 
+-> **Note:** This endpoint will always treat intentions with
+`Permissions` defined as *deny* intentions during evaluation, as this
```

Comment:
This could have been just for the UI, but I thought @freddygv was changing it so that our endpoint(s) would return `allow`, `deny`, or `nil` so that we could be smarter about `nil` being an L7 intention with permissions. Maybe not the case for this specific set though! Would probably be a nice improvement for the future it that's not the case right now.

Reply:
@freddygv  was just changing how the data was fetched for use in the topology endpoint. The existing endpoint was not changed.